mandatory/2-function-creation.js

```diff
@@ -6,7 +6,10 @@ Complete the function to check if the variable `num` satisfies the following req
 Tip: use logical operators
 */
 
-function validate(num) {}
+function validate(num) {
+  return (typeof num === "number" && num 2 === 0 && num <= 100);
```

Comment:
I think the num 2 === 0 should have been num 2%0 to check for even numbers? Perhaps I am getting this wrong, but I do believe that num 2 === 0 is not a valid expression. Maybe it was supposed to be num === 0